client/src/components/Add/asyncValidate.js

```diff
@@ -0,0 +1,13 @@
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
what does this file do?

Reply:
you should put an explanation in the PR description, QA shouldn't have to ask these things.

Would be easier if your issues were more clearly planned out...

Reply:
its a unused file we remove it :+1: